src/components/screens/PointOfInterest.js

```diff
@@ -16,12 +16,14 @@ import { ImagesCarousel } from '../ImagesCarousel';
 import { OrientationContext } from '../../OrientationProvider';
 import { useMatomoTrackScreenView } from '../../hooks';
 import { matomoTrackingString } from '../../helpers';
+import { WebViewMap } from '../map';
```

Comment:
![Bildschirmfoto 2020-12-09 um 21 28 57](https://user-images.githubusercontent.com/1942953/101683894-e00d2d80-3a65-11eb-9aa2-12d8a7b760f0.png)

sometimes this can happen, where an explicit import is necessary. sorry if i missdirected something here.

done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/137/commits/11f10e4f71a1949fa735808e88ca329e1f050f6f